_episodes/05-questions.md

```diff
@@ -13,6 +13,61 @@ keypoints:
 - "A well detailed research question can be answered with a good chart." 
 ---
 
+## Gapminder Ignorance Project
+
+[Gapminder][about-gapminder] is a "fact tank" from Sweden that maintains data and teaching resources about global development. Gapminder promotes a fact-based
+world view from a deeper understanding of reliable statistics. From their data,
+we can begin to formulate some research questions to explore:
+
+What are the major trends in global populations and economies?
+
+How is the world population distributed among countries and continents?
+
+How is population linked to economies and health?
+
+Has that changed over time?
+
+## Good Research Questions
+
+Good research questions are falsifiable/verifiable with data. They are well-defined and have a clear research outcome or set of outcomes.
+
+## Research questions lead to clear data visualizations.
+
+Data visualization through tables and graphics are a scientists way of conveying
+a convincing answer to their research question based on quantitative evidence.
+We focus on graphics, or __charts__, in this lesson for data visualization.
+
+Charts can take on endless forms and customizations to address your research question. We will work to simplify your options to effectively determine the *best* visualization for your data.
+
+## Defining your research question for visualization
+
+Dr. Andrew Abela from Catholic University of America and 'Extreme Presentation'
```

Comment:
1. Link to Abela's home page?
2. I think "Extreme Presentation" is Abela's company, so probably more accurate to say "Dr. Andrew Abela from the Catholic University of America put together..."